content/concepts/introduction/intro-to-consensus.md

```diff
@@ -16,22 +17,21 @@ This protocol is still evolving, as is our knowledge of its limits and possible
 
 ## Background
 
-Consensus protocols are a solution to the _double-spend problem_: the challenge to prevent someone from successfully spending the same digital money in two places. For a detailed analysis of the double-spend problem, how the XRP Ledger Consensus Protocol solves this problem, and the tradeoffs and limitations involved, see [Consensus Principles and Rules](consensus-principles-and-rules.html).
+Consensus protocols are a solution to the _double-spend problem_: the challenge to prevent someone from successfully spending the same digital money in two places. The hardest part about this problem is putting transactions in order: without a central authority, it can be difficult to resolve disputes about which transaction comes first when you have two or more mutually-exclusive transactions sent around the same time. For a detailed analysis of the double-spend problem, how the XRP Ledger Consensus Protocol solves this problem, and the tradeoffs and limitations involved, see [Consensus Principles and Rules](consensus-principles-and-rules.html).
```

Comment:
Rome, overall this is excellent documentation. What follows is a bunch of small recommendations. 

Here, I wonder about your description of the double spending problem. Does place really matter?

> < prevent someone from successfully spending the same digital money in two places
> ---
> > prevent someone from successfully spending the same digital money **twice**

Reply:
1. :+1: 
2. Technically no, and changed. However, attempting to spend in different places _does_ make it much easier to spend the same money twice.
3. Tricky question. The simplest example is the case where you have a balance that's not allowed to go negative; if you have two transactions that each spend all your money to a different person, those two transactions are mutually-exclusive: only one of them can be successful. The more complicated answer is that it depends on the rules of what transactions are allowed (more importantly, not allowed) to do, which is partly a matter of preference and partly situational.